### DISCUSSION TITLE
Question: Ability to access subject of the email from MessageView

### DISCUSSION THREAD

**Discussion by gencalda** [InboxSDK/InboxSDK#1115] [Q&A] [answered]
**Question(s)**

Hi. We have a requirement where it's necessary to include the subject of the email. Would it be possible to add the subject from the MessageView as an enhancement? I noticed that the subject of the thread is accessible from the ThreadView, but our need is for the actual subject of the message, not the subject of the ThreadView. Thank you for your consideration.

**Comment by jfmaggie** [comment, marked as answer]
hey @gencalda, not 100% sure what your requirement is and why you'd need messageView to access subject directly, we'd need more information around the ask here. [MessageView](https://inboxsdk.github.io/inboxsdk-docs/conversations/#messageview) itself could have different view state as Gmail visually display messages in a thread in different ways depending on what they are trying to show a user. 

That being said, you can get the subject via threadView with `messageView.getThreadView().getSubject()` if that's what you were looking for. :)
